### ISSUE TITLE
PS5 accepts connection but steamdeck does not do anything. 

### ISSUE THREAD

**Issue by matorre2048** [streetpea/chiaki4deck#27]
Hello, I have tried to get advice on this on reddit and other locations but i can't seem to gain any insight. 

I installed Chiaki using the chiaki4deck guide. When i run chiaki4deck on desktop mode, the PS5 displays a message that it has begun a streaming session, but the SteamDeck doesn't really do anything. It opens the window that i assume is meant for the streaming display but it is always black. Even if i leave it for a while it stays like that. The app becomes unresponsive if i try and close the window or the chiaki4deck application in general. I have to do a hard reboot to gain control of the system again. 

Similar is true for the game mode launch. When launching the chiaki4deck app (after configuring it as the guide recommends), the launch never moves beyond the screen for steamdeck where the steam logo is pinwheeling. Again, in order to regain any control over the unit i have to perform a hard reset.

**Comment by matorre2048** [comment]
I have tried re-installing chiaki4deck and it seems the issue that is happening is the following

```
No configuration found for 'Miguel\x2019s PS5'
```

Anyone have any advice for addressing this error?

**Comment by streetpea** [comment]
Is that \x in your PS5 name? There may be some issue with parsing that \x somewhere. Can you go to your PlayStation and check that the nickname you have set there is the same as what’s displayed in the configuration error? 

You can also run the following command on your Steam Deck to see what is has set for the name and see if you have a mismatch:

```
flatpak run re.chiaki.Chiaki4deck list
```

**Comment by matorre2048** [comment]
Thank you for pointing that out. 

The consoles name was set as 'Miguel's PS5' It seems that the configuration file recorded the unicode of the 'Right-Single-Quote' which caused the error. 

 When I ran chiaki4deck again the updated console name was there, but the same error existed. I have renamed the PS5 to remove the single quote and updated the configuration file to remove it as well. so that worked out. 
 
 I am now able to run the script and boot it from the Game Mode as well! thank you for your hard work with all this.
